src/app/clusters/chime-server/ChimeCluster.cpp

```diff
@@ -39,33 +42,50 @@ namespace chip {
 namespace app {
 namespace Clusters {
 
-ChimeServer::ChimeServer(EndpointId endpointId, ChimeDelegate & delegate) :
-    AttributeAccessInterface(MakeOptional(endpointId), Chime::Id), CommandHandlerInterface(MakeOptional(endpointId), Chime::Id),
-    mDelegate(delegate), mSelectedChime(0), mEnabled(true)
+ChimeCluster::ChimeCluster(EndpointId endpointId, ChimeDelegate & delegate) :
+    DefaultServerCluster({ endpointId, Chime::Id }), mDelegate(delegate)
 {
-    mDelegate.SetChimeServer(this);
+    mDelegate.SetChimeCluster(this);
 }
 
-ChimeServer::~ChimeServer()
+ChimeCluster::~ChimeCluster()
 {
-    // null out the ref to us on the delegate
-    mDelegate.SetChimeServer(nullptr);
-
-    // unregister
-    TEMPORARY_RETURN_IGNORED CommandHandlerInterfaceRegistry::Instance().UnregisterCommandHandler(this);
-    AttributeAccessInterfaceRegistry::Instance().Unregister(this);
+    Shutdown();
```

Comment:
The constructor doesn't call Startup, so we assume it was called in the usage of the cluster. Should we also assume this for the Shutdown? I know it doesn't do much but just for the sake of consistency.